lib/src/loader/loader.component.scss

```diff
@@ -0,0 +1,111 @@
+@import "~@sebgroup/bootstrap/scss/core";
+@import "~@sebgroup/bootstrap/scss/styles/spinners";
+
+@mixin setLoaderSize($size) {
+    > svg {
+        width: $size;
+        height: $size;
+    }
+}
+
+$offset: 200;
+$duration: 1.5s;
+$duration-sq: 1.2s;
+$loader-xs: 1rem;
+$loader-sm: 2.125rem;
+$loader-md: 4.063rem;
+$loader-lg: 6.25rem;
```

Comment:
![image](https://user-images.githubusercontent.com/49055952/86312135-9e758c80-bc54-11ea-8b70-4b944b9cb781.png)
Please use the existing SEB Bootstrap variables where possible